### PR TITLE
Prinz

### PR DESCRIPTION
**Pull request by prinz0147**

### CODE REVIEW

**Review by Lobason — COMMENTED** [review]
Hey Prinz who are you voting for in 2023, Obi or Tinubu

**Review by nkwatoh1 — COMMENTED** [review]
perfect work. we almost there

File: studentstories [hunk 1]
```diff
@@ -0,0 +1,2 @@
+My name is Prinz and i love postgress. Can definately be a pain at times but it can also be worth it.
+Few of my favorites are, adding users, editing files/directories and so on. Thanks :)
```
Comment (frubruce) [inline studentstories:2]: Hey Prinz, where are u originally from?

**Review by asahndah — COMMENTED** [review]
hey Prinz. iam excited for you. this is interesting

**Review by asahndah — APPROVED** [review]
hey Prinz. iam excited for you. this is interesting

**Review by nkwatoh1 — APPROVED** [review]

**Review by PhilipNze — APPROVED** [review]
Prinz continue doing want you known bro

**Review by Ta-tech-solution — COMMENTED** [review]
o ya merge it...hahahaha

**Review by Ta-tech-solution — APPROVED** [review]

**Review by Lobason — APPROVED** [review]

**Review by frubruce — APPROVED** [review]

**Comment by Ta-tech-solution** [comment]
on line 1 , you spelled your postgres with double 'S' instead of 1 'S'

**Review by kbih — APPROVED** [review]

**Pull request merged**